src/Umbraco.Cms.Api.Management/Services/ConflictingRouteService.cs

```diff
@@ -19,7 +20,7 @@ public class ConflictingRouteService : IConflictingRouteService
     public bool HasConflictingRoutes(out string controllerName)
     {
         var controllers = _typeLoader.GetTypes<ManagementApiControllerBase>().ToList();
-        foreach (Type controller in controllers)
+        foreach (Type controller in CollectionsMarshal.AsSpan(controllers))
```

Comment:
This is pointless, JIT will lower this.

Reply:
No, check the updated CollectionsBenchmark https://github.com/umbraco/Umbraco-CMS/pull/17106/files#diff-a65c2dcf45f047e0a7232b262aaa4a5dc4f21734c4a7a5912d426158464bf4fb
Or: https://www.youtube.com/watch?v=jUZ3VKFyB-A&t=640s
Or: https://www.linkedin.com/pulse/collectionsmarshalasspan-overhead-pgo-henrik-gedionsen/